model/project_vars.go

```diff
@@ -159,7 +159,7 @@ func FindOneProjectVars(ctx context.Context, projectId string) (*ProjectVars, er
 		return nil, err
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), defaultParameterStoreAccessTimeout)
+	ctx, cancel := context.WithTimeout(ctx, defaultParameterStoreAccessTimeout)
```

Comment:
I think we could get rid of the context timeout here, because it was only used in the absence of a passed-down context.

Reply:
SGTM!